osbs/api.py

```diff
@@ -617,6 +617,7 @@ def _do_create_prod_build(self, git_uri, git_ref,
             scratch=self.build_conf.get_scratch(scratch),
             reactor_config_secret=self.build_conf.get_reactor_config_secret(),
             reactor_config_map=self.build_conf.get_reactor_config_map(),
+            reactor_config_yaml=self.build_conf.get_reactor_config_yaml(),
```

Comment:
This shouldn't be in `build_conf`, it should be a parameter like `koji_task_id`, for example.

Reply:
ok